test/00_suite/test_bitbank_order.py

```
@@ -0,0 +1,109 @@
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)